builtin/providers/kubernetes/patch_operations.go

```diff
@@ -0,0 +1,135 @@
+package kubernetes
+
+import (
+	"encoding/json"
+	"reflect"
+	"sort"
+	"strings"
+)
+
```

Comment:
Nit pick - should this file really be called patch_operations? It seems there are patch and replace operation funcs within :)